### PR TITLE
Rewrite doc

### PR DESCRIPTION
**Pull request by Serhio77777**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,14 +10,20 @@ Setup (required Node 4.x)
 
     yarn
 
-Run a dev server
+    yarn postinstall
```
Comment (kikill95) [inline README.md]: Why?

Comment (Serhio77777) [reply README.md]: I want to add bower install command. But there are the line at the scripts at the package.json: "postinstall": "bower install && gulp"...

File: package.json [hunk 2]
```diff
@@ -1,8 +1,10 @@
 {
   "name": "ionickit",
-  "version": "1.1.0",
+  "version": "0.25.3",
```
Comment (kikill95) [inline package.json]: 1.1.1, please

File: package.json [hunk 3]
```diff
@@ -30,5 +32,10 @@
   },
   "repository": "git@github.com:ekreative/ionic-starter-kit.git",
   "author": "Light <kirill.gusyatin@ekreative.com>",
-  "license": "MIT"
+  "license": "MIT",
+  "cordova": {
```
Comment (kikill95) [inline package.json]: add ios as well, please

**Review by kikill95 — APPROVED** [review]

**Pull request merged**